CMakeLists.txt

```diff
@@ -42,15 +42,15 @@ endif ()
 set(MMSEQS_CXX_FLAGS "-std=c++0x")
 if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm.*|ARM.*)")
     set(ARM 1)
-    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -DNEON=1 -DSSE=1 -fsigned-char -mfpu=neon")
+    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -fsigned-char -mfpu=neon")
 elseif (CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*)")
     set(ARM 1)
-    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -DNEON=1 -DSSE=1 -fsigned-char")
+    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -fsigned-char")
 elseif (CMAKE_SYSTEM_PROCESSOR MATCHES "PPC64*|ppc64*|powerpc64*")
     set(PPC64 1)
-    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -DSSE=1 -fsigned-char -std=gnu++0x -mcpu=power8 -mvsx")
+    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -fsigned-char -std=gnu++0x -mcpu=power8 -mvsx")
```

Comment:
I think `-mcpu=power8 -mvsx` should be set in CI config, or elsewhere, but not for all PPC64 users